client/app/services/query-result.js

```diff
@@ -102,6 +102,9 @@ function QueryResultService($resource, $timeout, $q) {
             } else if (isString(v) && v.match(/^\d{4}-\d{2}-\d{2}$/)) {
               row[k] = moment.utc(v);
               newType = 'date';
+            } else if (v.match(/^\d{4}\d{2}\d{2}$/)) {
```

Comment:
Unless I'm missing something, this regex has potential for a lot of false detections. Basically any numeric column in the range of 10000000-99999999 will be detected as dates, no?

Reply:
Yes, you're right. I took the regex used above and removed the dashes but that makes the range of error worse than above. I've updated with a new regex pattern that is a little more restrictive.